context/src/junit/validator.rs

```diff
@@ -39,6 +42,15 @@ pub enum JunitValidationIssue<SO, I> {
     Invalid(I),
 }
 
+impl<SO: fmt::Display, I: fmt::Display> fmt::Display for JunitValidationIssue<SO, I> {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        match self {
+            Self::SubOptimal(i) => write!(f, "{i}"),
+            Self::Invalid(i) => write!(f, "{i}"),
+        }
+    }
+}
```

Comment:
Trunk Check was failing on a Clippy lint that disallows direct implementation of the `ToString`:
https://rust-lang.github.io/rust-clippy/master/index.html#to_string_trait_impl

It requires implementing `Display` instead which I did so in a generic way here so that we don't need to repeat ourselves below for each validation enum type